src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,63 @@
+$weekday:
```

Comment:
it can be in another file, but maybe optional

Reply:
Agree with @AmbeMO , create file for variables